doc/user-guide/alt-deploy.md

```diff
@@ -0,0 +1,66 @@
+## Deploying via olm
+
+- Clone the repo
+```
+git clone git@github.com:red-hat-storage/lvm-operator.git && cd lvm-operator
+```
+
+### Deploying pre-built images
+
+- Quick install using pre-built container images via OLM.
+
+```
+make deploy-with-olm
```

Comment:
This requires CATALOG_IMG to be set.

Reply:
- this was for pre-built images and as there's no pre-built images, will remove this section.